.ci/project-dependencies.yaml

```diff
@@ -60,14 +60,16 @@ dependencies:
   - project: kiegroup/openshift-drools-hacep
     dependencies:
       - project: kiegroup/droolsjbpm-integration
+      - project: kiegroup/drools
+      - project: kiegroup/jbpm
```

Comment:
Wrong change: `openshift-drools-hacep` was correct, `jbpm` and `drools` should be added to `droolsjbpm-integration`

Reply:
done